src/main.py

```diff
@@ -349,12 +366,28 @@ def analyze_line(line: str) -> str:
         converted = f"|-singleturn|p{target_player+1}a: {target_mon.nick}|move: Roost"
 
     elif heal_pat.match(line):
-        move, use_player, target_player, use_mon, target_mon = moves_buffer
+        target_player = identify_player(line, heal_pat)
+        target_mon = players[target_player].currentmon
         target_mon.heal(50.0)
         converted = f"|-heal|p{target_player+1}a: {target_mon.nick}|{target_mon.approx_hp()}\/100{target_mon.space_status()}"
 
+    elif status_cleared_lum_pat.match(line):
+        target = identify_player(line, status_cleared_lum_pat)
+        target_mon = players[target].currentmon
+        if target_mon.status != utils.Status.NONE:
+            target_mon.status = utils.Status.NONE
+            converted = f'|-curestatus|p{target + 1}a: {target_mon.nick}|{target_mon.status_string()}|[msg]'
+        else:
+            converted = f"|-end|p{target + 1}a: {target_mon.nick}|confusion"
 
 
+    elif wish_pat.match(line):
+        if gen == 5:
+            raise Exception("Wish is not implemented for modern gen")
```

Comment:
Replace message with "Wish is not supported for gen 5 logs."

Reply:
I think not implemented is better but I've changed modern gen to gen 5 logs